### DISCUSSION TITLE
[@types/levelup][@types/abstract-leveldown] LevelUp interface is missing properties `status` & `isOperational()`

### DISCUSSION THREAD

**Discussion by istherepie** [DefinitelyTyped/DefinitelyTyped#57662] [Issues with a @types package] [answered]
Greetings!

I noticed that `status` and `isOperational()` is missing from the LevelUp interface:

From the docs:
* https://github.com/Level/levelup/blob/master/README.md#dbstatus
* https://github.com/Level/levelup/blob/master/README.md#dbisoperational

Example: 

```typescript

import level from "level";

const db: level.LevelDB<any, any> = level("testdb")

let t1: string = db.status
console.log(t1)

let t2: boolean = db.isOperational()
console.log(t2)


```

The code above will result in the following errors:

```shell

> tsc ./src/main.ts --esModuleInterop

src/main.ts:5:21 - error TS2339: Property 'status' does not exist on type 'LevelDB<any, any>'.

5 let t1: string = db.status
                      ~~~~~~

src/main.ts:8:22 - error TS2339: Property 'isOperational' does not exist on type 'LevelDB<any, any>'.

8 let t2: boolean = db.isOperational()
                       ~~~~~~~~~~~~~


Found 2 errors.

```

However this functionality is part of the public api,

* from [level/abstract-leveldown](https://github.com/Level/abstract-leveldown/blob/ff6d9556186de725cd2028204612c36100808e28/abstract-leveldown.js#L15)
```javascript
function AbstractLevelDOWN (manifest) {
  this.status = 'new'

```

* from [level/abstract-leveldown](https://github.com/Level/abstract-leveldown/blob/ff6d9556186de725cd2028204612c36100808e28/abstract-leveldown.js#L368)

```javascript
AbstractLevelDOWN.prototype.isOperational = function () {
  return this.status === 'open' || this._isOperational()
}
```

* from [level/levelup](https://github.com/Level/levelup/blob/46d6e183d1d93404417c107f68698654ba6aca52/lib/levelup.js#L92):

```javascript
LevelUP.prototype.isOperational = function () {
  return this.db.status === 'open' || this.db.status === 'opening'
}
```

[a comment by an automated account is not shown]

**Comment by carsonfarmer** [comment]
Yup, looks like those methods are missing! Feel free to submit a PR to add those, I will endeavor to review as quickly as possible.

**Reply by istherepie** [reply]
Thanks for the swift reply - I have already done so - please see the PR #57663!
